content/en/docs/Installation/installation-guide/prerequisites.md

```diff
@@ -48,17 +47,25 @@ supported Istio versions.
 
 <br />
 
+## Openshift Console Plugin (OSSMC) Version Compatibility
+
+Currently OSSMC plugin only works with Kiali server 1.73.
+
+{{<compat-table-ossmc>}}
+
+<br />
+
 ## OpenShift Service Mesh Version Compatibility
 
 {{% alert title="OpenShift" color="warning" %}}
 If you are running Red Hat OpenShift Service Mesh (RH OSSM), use only the bundled version of Kiali.
```

Comment:
I've never seen Red Hat documentation or other material show this as "RH OSSM" - it is always just "OSSM" as far as I've ever seen. I would take out the RH here.

```suggestion
If you are running Red Hat OpenShift Service Mesh (OSSM), use only the bundled version of Kiali.
```